Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/submission/SubmissionQRCodeScanFragment.kt

```diff
@@ -134,6 +134,11 @@ class SubmissionQRCodeScanFragment : BaseFragment() {
         })
 
         viewModel.registrationState.observeEvent(viewLifecycleOwner, {
+            binding.submissionQrCodeScanSpinner.visibility = when (it) {
```

Comment:
we access the binding here instead of in the view model because of the event wrapper around the live data

Reply:
Don´t we usually set visibility using formatters in xml? Is this not possible in this case?

Reply:
@kolyaopahle could update us on this one?